### PR TITLE
Tutorial for Sheets get latitude

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
Formatted latitude tutorial. I decided to just move the content into a PR since comments etc were becoming hard to track and respond to.

### CODE REVIEW

File: tutorials/sheets_latitude.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+![](/assets/images/tutorials/sheets_latitude_tutorial_4.png)
+
+Next, you’ll want to retrieve the country names and position them conveniently near these DCIDs. You can use the plugin function [`DCGETNAME`](/api/sheets/get_name.html) to access this information. Type `=dcgetname(B1:B)` in cell C1. Your final output will look like this:
+
+![](/assets/images/tutorials/sheets_latitude_tutorial_5.png)
+
+## Step 3: Populating the spreadsheet with capital and latitude information
+In this step, you will obtain all South American countries’ capitals and latitudes. To do this, you will need to get the value of the property names `latitude` and [`administrativeCapital`](https://datacommons.org/browser/administrativeCapital) for each country on the date specified. (As an aside, if you'd like to see what properties are available for any given entity, Data Commons provides a [graph browser](https://datacommons.org/browser) tool enabling you to look up any entity in the graph and view its associated properties.) You can use the [`DCPROPERTY`](/api/sheets/get_property.html) method to do this.
```
Comment (beets) [inline tutorials/sheets_latitude.md:36]: consider also linkifying "latitude"
```suggestion
In this step, you will obtain all South American countries’ capitals and latitudes. To do this, you will need to get the value of the property names [`latitude`](https://datacommons.org/browser/latitude) and [`administrativeCapital`](https://datacommons.org/browser/administrativeCapital) for each country on the date specified. (As an aside, if you'd like to see what properties are available for any given entity, Data Commons provides a [graph browser](https://datacommons.org/browser) tool enabling you to look up any entity in the graph and view its associated properties.) You can use the [`DCPROPERTY`](/api/sheets/get_property.html) method to do this.
```

Comment (beets) [reply tutorials/sheets_latitude.md:36]: would it also make sense to add a link to the browser node for southamerica?
https://datacommons.org/browser/southamerica

File: tutorials/sheets_latitude.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Type `=DCPROPERTY(B1:B, "administrativeCapital")` into cell D1. Your output should look like this:
+
+![](/assets/images/tutorials/sheets_latitude_tutorial_6.png)
+
+Now, to get the latitude of each capital, type `=DCPROPERTY(D1:D, "latitude")` into cell E1.
+
+![](/assets/images/tutorials/sheets_latitude_tutorial_7.png)
+
+If you’d like to experiment more with the data and methods from this tutorial, check out <https://docs.google.com/spreadsheets/d/1AYOD9yX59aKNNHoLLsmy00uihU72EKDFDZVo5KdX7H0/edit#gid=0>. You can clone the sheet by clicking on the File dropdown, then choosing the 'Make a copy' option.
```
Comment (beets) [inline tutorials/sheets_latitude.md:46]: as discussed, could you please add a note to enable the api after making a copy?

**Review by beets — COMMENTED** [review]
thanks - it's looking good. just a couple of minor comments

could you please also add me as an editor to the reference sheet? thanks!

File: tutorials/sheets_latitude.md (hunk 2 above)
Comment (KilimAnnejaro) [reply tutorials/sheets_latitude.md:46]: done!

File: tutorials/sheets_latitude.md (hunk 1 above)
Comment (KilimAnnejaro) [reply tutorials/sheets_latitude.md]: I linked to the DCID not in this paragraph, but in step 2. Lmk what you think!

**Review by beets — APPROVED** [review]
looks good!

**Pull request merged**